src/frontends/jax/src/op_table.cpp

```diff
@@ -71,8 +72,11 @@ const std::map<std::string, CreatorFunction> get_supported_ops_jaxpr() {
             {"ge", op::translate_binary_op<v1::GreaterEqual>},
             {"gt", op::translate_binary_op<v1::Greater>},
             {"integer_pow", op::translate_integer_pow},
+            {"lt", op::translate_binary_op<v1::Less>},
+            {"le", op::translate_binary_op<v1::LessEqual>},
             {"max", op::translate_1to1_match_2_inputs<v1::Maximum>},
             {"mul", op::translate_1to1_match_2_inputs<v1::Multiply>},
+            {"neg", op::translate_negative},
```

Comment:
please use `op::translate_1to1_match_1_input` as for other unary operations translated 1-to-1. So you don't need separate sources with translator